Add more implementer details.

BigBlueHat opened the pull request:
- Add .editorconfig.
- Fix whitespace in GrotoNetworking.json.
- Add more metadata to implementations.

BigBlueHat committed:
Add .editorconfig.

BigBlueHat committed:
Fix whitespace in GrotoNetworking.json.

BigBlueHat committed:
Add more metadata to implementations.

aljones15 commented:
can this be tested with a few test suites first? Properties in a manifest are often cast to the Endpoint class so adding new properties might result in some small correctable issues.

BigBlueHat commented:
@aljones15 yeah, there are also reporter updates to make. Mainly looking for format/shape feedback first.

BigBlueHat committed:
Add initial context file.

This lays the foundation for eventually turning this metadata into VCs.

The context uses a mix of the W3Cs EARL vocabulary, DOAP (which EARL
uses), Schema.org, and a temporary `reporter` namespace mapped to
https://github.com/digitalbazaar/mocha-w3c-interop-reporter#
which is the primary software which consumes these details to generate
W3C test reports.

This context MAY (and likely should) move in the future either to that repo or
to some more permanent home.

BigBlueHat committed:
Set types using new context terms.

BigBlueHat committed:
Add support email address to Digital Bazaar info.

davidlehn reviewed (commented):
It might be good to find an EARL/DOAP expert if the idea is to properly align with those schemas.  For instances, there is some mixing of schema.org and doap Person and Organization, which doesn't align with the DOAP schema ranges.  In this use case, it probably doesn't matter.  Not sure what best practices are with that.

Also the usage here isn't quite like what JSON-LD and rdf-canon use in their test results.  Again, it might not matter here.  This is headed the right direction in any case and will help improve reports.

BigBlueHat commented:
Here's how JSON-LD implementations were described: https://w3c.github.io/json-ld-api/reports/#instructions-for-submitting-implementation-reports

aljones15 reviewed (approved):
Approving as this is needed and an excellent idea, just hope it doesn't require to many changes to sister libs to accomplish this PR.

lanterno commented:
Hello @BigBlueHat , this is a great addition here. It would help a lot in figuring out which implementations have open-source projects that we can reuse. I hope you haven't abandoned this PR 🙏 

btw, would it make sense to add a flag to indicate which implementation is open source (or not) ? 
(Right now, I figure it out by following the github.com/xx/xx links, but an explicit flag might be more interesting)

TallTed commented:
I'll be happy to review this (not sure why I didn't see the request last year), after the conflicts are resolved.
